42mad

```diff
@@ -184,6 +184,23 @@ echo "Setting android settings for workers"
 ! settings get secure location_providers_allowed|grep -q gps && settings put secure location_providers_allowed +gps
 echo "Successfully set the settings for workers"
 }
+#Not tested yet
+set_mac_unify(){
+echo "Setting Mac in unifykey"
+
+if [[ "$(ifconfig eth0|awk '/HWaddr/{print $5}')" == "00:15:18:01:81:31" ]] ;then
```

Comment:
this check used to trigger it because a95x used to get this mac on all... but now its actually random. i dont expect this check to trigger now

Reply:
this is excellent thank you. its going to PR limbo for a bit because bananas building an api endpoint to store / retrieve the mac in the db which will be part of this.